ote/src/cljs/ote/views/main.cljs

```diff
@@ -42,6 +42,8 @@
                    :primary-text user-name
                    :selected-text-color (color :grey900)
                    :on-click #(e! (fp-controller/->ChangePage :front-page))} ]
+    [ui/menu-item {:primary-text "Näytä profiili"
```

Comment:
Use translations in menu item texts.
Use an event for the click handler.

Reply:
Translated, but I didn't add correct event yet. 
Profile link is different for different users. And I need to investigate it a bit more.